cylc/flow/rundb.py

```diff
@@ -161,18 +161,18 @@ def add_update_item(self, set_args, where_args):
         self.update_queues[stmt].append(stmt_args)
 
 
-class CylcSuiteDAO:
-    """Data access object for the suite runtime database."""
+class CylcWorkflowDAO:
+    """Data access object for the workflow runtime database."""
 
     CONN_TIMEOUT = 0.2
     DB_FILE_BASE_NAME = "db"
     MAX_TRIES = 100
-    RESTART_INCOMPAT_VERSION = "8.0a2"  # Can't restart suite if <= this vers
+    RESTART_INCOMPAT_VERSION = "8.0a2"  # Can't restart if <= this version
```

Comment:
Just realised, the changes to the DB table names will mean anything last run on 8.0b1 or below won't restart.
```suggestion
    RESTART_INCOMPAT_VERSION = "8.0b1"  # Can't restart if <= this version
```

Reply:
Yep, good spot.